.github/workflows/test.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.17.x, 1.18.x]
+        go-version: [1.20.x, 1.21.x]
```

Comment:
should I bump the lint version too?  v1.20?

Reply:
latest pls

Reply:
I think you can just use `1.x`